run_page/utils.py

```diff
@@ -92,7 +92,7 @@ def get_strava_last_time(client, is_milliseconds=True):
         return 0
 
 
-def upload_file_to_strava(client, file_name, data_type, force_to_run=True):
+def upload_file_to_strava(client, file_name, data_type, force_to_run=False):
```

Comment:
The default `force_to_run` shouldn't be changed here, as it will affect features like Garmin, Keep, and Nike.
Just change the param in `tcx_to_strava_sync`.

Or just keep this unchanged, since this repo is mainly for running. 

I'll change the `force_to_run=False` in workouts_page.

Reply:
Ok, I see